apps/user-office-frontend-e2e/cypress/integration/generic_templates.ts

```diff
@@ -32,6 +32,11 @@ context('GenericTemplates tests', () => {
   let workflowId: number;
   let createdQuestion1Id: string;
 
+  beforeEach(() => {
```

Comment:
I've noticed that if you have `before hook` in context AND in suite then they might cause a race conditions. Last week we made it so to only have `before hook` in the suite

![image](https://user-images.githubusercontent.com/58165815/196994944-eddd6708-034f-40ff-9fc5-4bfdfe707807.png)